docs/src/content/docs/api/webhooks.mdx

```diff
@@ -0,0 +1,492 @@
+---
+title: "Webhooks"
+sidebar:
+  order: 60
+tableOfContents: true
+---
+import { Badge, Tabs, TabItem } from '@astrojs/starlight/components';
+import CallOut from '@components/CallOut.astro';
+
+<CallOut
+  title="API Key Required"
+  text="The Ultravox API requires an API key. For more details see "
+  link={{ url: "../auth", text: "Authentication." }}
+/>
+<br />
+
+Webhooks allow you to receive real-time notifications about events in your Ultravox account. You can configure webhooks to send HTTP POST requests to a specified URL when certain events occur.
+
+## Webhooks in Ultravox
+When creating a webhook, you must provide a URL that will receive the webhook notification. Your service must return an HTTP status code of 200 ("OK"). If your service fails to respond with a 200, we will retry sending again.
```

Comment:
Any 200-class response code is fine (or 300-class for that matter).  Just not a 400- or 500-class.  (For example, return 204 No Content probably makes the most sense in general and would be totally fine.)

Reply:
cool. adding this.